NitroxServer/GameLogic/EventTriggerer.cs

```diff
@@ -42,7 +43,8 @@ public EventTriggerer(PlayerManager playerManager, double elapsedTime, double? a
             }
             else
             {
-                AuroraExplosionTime = RandomNumber(2.3d, 4d) * 1200d * 1000d; //Time.deltaTime returns seconds so we need to multiply 1000
+                DeterministicGenerator generator = new(seed, nameof(EventTriggerer));
+                AuroraExplosionTime = generator.NextDouble(2.3d, 4d) * 1200d * 1000d; //Time.deltaTime returns seconds so we need to multiply 1000
```

Comment:
This line will change to have elapsedTimeOutsideStopWatch added to the generated random number.
![SetExplodeTime](https://user-images.githubusercontent.com/24827220/146689740-2bdffcdd-b97d-4f5b-898a-a5a3aa32930b.png)